x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/tabs/alerts/alerts_tab_content.tsx

```diff
@@ -47,11 +43,7 @@ export const AlertsTabContent = () => {
 
   const { alertsTableConfigurationRegistry, getAlertsStateTable: AlertsStateTable } =
     triggersActionsUi;
-
-  const hostsWithAlertsKuery = hostNodes
-    .filter((host) => host.alertsCount)
-    .map((host) => `"${host.name}"`)
-    .join(' OR ');
+  const hostsWithAlertsKuery = hostNodes.map((host) => `host.name: "${host.name}"`).join(' OR ');
```

Comment:
The name is somewhat misleading; WithAlertsKuery indicates that there is an additional query specifically for the alerts. Is this the case?

Reply:
The idea was to show the all `hosts` that have `Alerts` as `kuery` filter inside the alerts page - so it is there to filter all hosts that have alerts (same filter we have for the count) that's why I named it like that but you are right that it can be misleading and lead to the assumption that there are some alert filters included.

I am wondering how to name it to be more explicit - I changed it to `hostNamesKuery` maybe it fits better, wdyt?

Reply:
Thanks, sounds good to me 🙏